### PR TITLE
PRIORITY - Adding logo refs for Filecoin icons

### PR DESCRIPTION
**Pull request by eshon**
This enables the Filecoin logo to display on chainlist.org

Tested with a local build and the Filecoin logos display once these are added.

re: https://github.com/DefiLlama/chainlist/issues/310

Filecoin is launching a new EVM next Tuesday on Pi Day and it'd be great if the Filecoin logos can show up on chainlist.org. @0xngmi @realdealshaman

### CODE REVIEW

**Comment by eshon** [comment]
Is it possible to review and merge this soon @0xngmi @realdealshaman? Filecoin is launching a new VM next Tuesday on March 14th and we'd like to have the logos visible on chainlist.org.

File: constants/chainIds.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
   "250": "fantom",
   "269": "hpb",
   "288": "boba",
+  "314": "filecoin",
```
Comment (0xngmi) [inline constants/chainIds.json:34]: that's 6 chains?

Comment (0xngmi) [reply constants/chainIds.json:34]: afaik there should be a single chain here, if the other ones are testnets we shouldn't tag them as filecoin

Comment (0xngmi) [reply constants/chainIds.json:34]: only mainnet

**Comment by 0xngmi** [comment]
pushed a fix for this on 60c4fe1

File: constants/chainIds.json (hunk 1 above)
Comment (eshon) [reply constants/chainIds.json:34]: Yes, the others are testnets.

Thank you for adding the mainnet one to  chainlist.org!

**Comment by raulk** [comment]
Thank you @0xngmi! @eshon it looks like the PR has developed conflicts. Could you take a look?

**Review by presata90 — APPROVED** [review]

**Comment by realdealshaman** [comment]
we've already added the logo for filecoin mainnet, we don't add testnets to the chainIds.json

**Pull request closed without merging**